widget/ui/src/components/ListItemButton/ListItemButton.styles.ts

```diff
@@ -6,6 +6,7 @@ export const BaseListItemButton = styled('button', {
   width: '100%',
   backgroundColor: 'transparent',
   fontFamily: 'inherit',
+  padding: '$0',
```

Comment:
Can we use `unset` or something? or directly using `0`?  I don't see any point to read a 0 from theme.